### ISSUE TITLE
feature request: customizing swipe speed

### ISSUE THREAD

**Issue by jusheen** [leecade/react-native-swiper#1230]
Hello! I was wondering if there could be an option to customize the speed at which the slides are swiped. Right now it seems like there is only an option to change the time in between swipes via the `autoplayTimeout` property, but not the actual swipe speed.

**Comment by shakawy** [comment]
did u use `autoplayTimeout` property ?

**Comment by jusheen** [comment]
@shakawy Yes, I did. I made a typo in my issue — I meant `autoplayTimeout` not `autoplay`.

**Comment by Rcoder0907** [comment]
I needed the same as well

**Comment by shakawy** [comment]
I still don't understand your request, did you mean gesture swipe speed?

**Comment by sp0re** [comment]
+1
sometimes we need it move slowly

**Comment by Ggayane** [comment]
+1
